x-pack/plugins/index_lifecycle_management/public/application/services/filter_items.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const filterItems = <T>(field: keyof T, filter: string = '', items: T[] = []): T[] => {
```

Comment:
I know this is pre-existing, but it seems weird to me that the `filter` param is optional 😅

Reply:
I agree, I made it mandatory and initialised filter in useState with an empty string.